backend/routes/RoutesCommon.go

```diff
@@ -0,0 +1,93 @@
+package routes
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"database/sql"
+
+	"github.com/ucsdscheduleplanner/UCSD-Schedule-Planner/backend/db"
+)
+
+// queryAndResponse is a function to
+// query using ds
+// process rows using rowScanner
+// write response
+func queryAndResponse(
```

Comment:
so typically when someone queries something they expect something back, (this is the response). So queryAndResponse is not needed, just a simple query is fine

Reply:
This function is for D.R.Y., i.e. to replace all the identical logic of each route. Are you suggesting that I should have 2 function: query() and response() for the sake of D.R.Y.?

Reply:
I am saying that when someone queries something, they will get a response.

For example when I query a database I expect to get a response. Say someone queries me for something, then I would query my database and then respond to whoever asked me. 

But I don't need to tell the person who is calling me that I have to query AND respond, when they query something from me, it is the expectation that I WILL respond, so there is no need to make the name queryAndResponse, just make it query.